clients/admin-ui/src/features/common/Breadcrumbs.tsx

```diff
@@ -1,13 +1,26 @@
-import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from "fidesui";
+import {
```

Comment:
Updated the breadcrumbs component. Added support for icons, expanded Link to be able to receive query params, added prop optionally be able to override some styles. Remove isOpaque props since it was too specific and it won't be used anymore. Removed onClick since it wasn't working nor being used.